docs-ref-services/servicebus.md

```diff
@@ -15,34 +15,189 @@ ms.service: service-bus
 
 ## Overview
 
-Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging. 
+Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging.
+
+As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves breaking changes.
+Please read [Migration from v0.21.1 to v0.50.0](#migration-from-v0.21.1-to-v0.50.0) to determine if upgrading is
+right for you at this time.
+
+For documentation on the legacy HTTP-based operations please see [Using HTTP-based operations of the legacy API](#using-http-based-operations-of-the-legacy-api)
+
+
+## Prerequisites
+
+* Azure subscription - [Create a free account](https://azure.microsoft.com/free/)
+* Azure [Service Bus namespace and management credentials](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal)
+* [Python 2.7-3.7](https://www.python.org/downloads/)
+
 
 ## Install the libraries
 ```bash
 pip install azure-servicebus
 ```
 
+## Create Client
+
+To get started, you will need to create a ServiceBusClient using the
+namespace and credentials that can be retrieved from the Azure Portal.
```

Comment:
All mentions of Azure portal should use lowercase 'p' in portal.

Having said that, is there a way to obtain this with a CLI command? If we can avoid sending people to the portal, we should.

Check the [Authentication](https://github.com/binderjoe/cosmos-python-prototype#authentication) section of the Cosmos README for an example of how it can be laid out.

Reply:
See my comment below about connecting the client.
Connection string seems to be the preferred method at this point - I don't think the CLI can return connection strings?